cityofaustin/atd-moped
# Add default milestones when creating projects

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/9102
and 
https://github.com/cityofaustin/atd-data-tech/issues/9258

## Testing
**URL to test:** 
 https://moped-test.austinmobility.io/moped/
moped-test or run locally

**Steps to test:**
Add a new project. From the "timeline template" dropdown, choose any of the four options. 
Upon creation, go to the project's timeline tab. Scroll down and see the milestones already populating the table. 

If you create a new project without selecting anything from the dropdown, the milestone table is empty. 


---
#### Ship list
